test/Common/HttpApiHandlerTests.cs

```diff
@@ -383,6 +385,67 @@ public async Task GetQueryStatus_is_Success()
             Assert.Equal("Running", actual[1].RuntimeStatus);
         }
 
+        [Fact]
+        public async Task GetQueryStatusWithPaging_is_Success()

```

Comment:
It looks like the new test did not run. I think it's because it is missing a required attribute. Please add the following:
```csharp
[Trait("Category", PlatformSpecificHelpers.TestCategory)]
```